SchoolXplorer.Application/Dtos/SchoolDistrictDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SchoolXplorer.Application.Dtos
+{
+	public class SchoolDistrictDto
+	{
+		public string? Id { get; set; }
```

Comment:
probably better to have separate dtos for create and update, instead of making Id nullable.

Reply:
sure